Sources/Amplitude/Timeline.swift

```diff
@@ -127,11 +160,53 @@ public class Timeline {
             if let mediator = plugins[type] {
                 mediator.plugins.forEach { (plugin) in
                     closure(plugin)
-                    if let destPlugin = plugin as? DestinationPlugin {
```

Comment:
@liuyang1520 did we need this for a customer? Is it ok we remove nested plugins?

Reply:
@yuhao900914 , feel free to correct me or comment here.

If it is a destination plugin, this ensures that the internal [`timeline`](https://github.com/amplitude/Amplitude-Swift/blob/2088f7619f1c2180f3e66e9f760c3374cce0abb3/Sources/Amplitude/Plugins/DestinationPlugin.swift#L11) (in destination plugin) will apply it. So I think it is needed here.

Reply:
Do we need this feature? We don't support nested plugins in other platforms. https://github.com/amplitude/Amplitude-Swift/pull/53/files/60010af8481f4636bafcd86add36685c6a0b7307#r1188002085

Reply:
Replied in the right thread.

Reply:
Reverted